Symfony/CS/Fixer/PSR2/SwitchCaseSpaceFixer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Symfony\CS\Fixer\PSR2;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+class SwitchCaseSpaceFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        $caseTokens = $tokens->findGivenKind(T_CASE);
+
+        foreach ($caseTokens as $caseIndex => $caseToken) {
```

Comment:
I think is a good idea to check all tokens from end to start, because if you remove/clear or add tokens the indexes of other tokens may change. If you fetch indexes first and then manipulate the collection the indexes fetched may no longer be what you expect.